.github/docker/Dockerfile.tmp_vllm

```diff
@@ -0,0 +1,414 @@
+# TODO(elainwy): remove this file after the torch nightly dockerfile is in sync in vllm repo
```

Comment:
Is there any particular part of this dockerfile that we should be paying attention to?